spec_linux.go

```diff
@@ -150,3 +153,16 @@ type Resources struct {
 	// Network restriction configuration.
 	Network Network `json:"network"`
 }
+
+type LinuxState struct {
+	State
+	// Linux holds platform specific state information for linux.
+	Linux struct {
+		// Namespaces holds paths to the container's namespaces on disk.
+		Namespaces map[string]string `json:"namespaces"`
+		// Cgroups holds paths to the container's cgroup paths.
+		Cgroups map[string]string `json:"cgroups"`
+		// ExternalFds holds paths to the container's STDIO.
```

Comment:
I'd replace “STDIO” here with “open file descriptors”.  You mention “STDIO and other…” in `runtime.md`, which is a nice hint, but I'd rather avoid being overly-specific in the Go-struct comments.